CatalogService/CatalogService/src/Domain/Domain.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net6.0</TargetFramework>
+        <RootNamespace>CatalogService.Domain</RootNamespace>
+        <AssemblyName>CatalogService.Domain</AssemblyName>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <PackageReference Include="MediatR" Version="9.0.0" />
```

Comment:
In clean architecture, domain projects should be dependent on external libraries only if it's completely necessary. IMO, dependency on such libs as MediatR should be in domain project under any circumstances.